pkg/metrics/metrics.go

```diff
@@ -0,0 +1,41 @@
+package metrics
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	log = logf.Log.WithName("metrics")
```

Comment:
It seems that `log` is unused in `metrics.go`, can we remove it?

Reply:
removed. thanks.